libnavigation-core/src/test/java/com/mapbox/navigation/core/replay/route/RemoveAcceleration.kt

```diff
@@ -0,0 +1,30 @@
+package com.mapbox.navigation.core.replay.route
```

Comment:
let's move it to another package 
```suggestion
package com.mapbox.navigation.core.testituls.replay
```

Reply:
done